test/test_data/apt_norms.csv

```diff
@@ -0,0 +1,9 @@
+parameter,mean,std
+main_deflector_parameters_theta_E,0.7992458031844867,0.15239538119453927
```

Comment:
I'm guessing this is pulled from an actual norm file. If you manually change the values to be more exact (0.8, 0.15, etc.) could be easier for testing.